packages/@o3r/eslint-config-otter/typescript.cjs

```diff
@@ -277,7 +277,10 @@ module.exports = {
     ],
     'jsdoc/check-access': 'warn',
     'jsdoc/check-alignment': 'error',
-    'jsdoc/check-examples': 'off',
+    // TODO: Activate when available on ESLint 8+
```

Comment:
is it worth creating an issue and adding it to the TODO so that we are sure we won't forget about it